ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Server.hs

```diff
@@ -73,32 +73,32 @@ chainSyncBlocksServer
     :: forall m blk. (IOLike m, HasHeader (Header blk))
     => Tracer m (TraceChainSyncServerEvent blk)
     -> ChainDB m blk
-    -> Reader m blk (WithPoint blk (Serialised blk))
+    -> Follower m blk (WithPoint blk (Serialised blk))
     -> ChainSyncServer (Serialised blk) (Point blk) (Tip blk) m ()
-chainSyncBlocksServer tracer chainDB rdr =
+chainSyncBlocksServer tracer chainDB flr =
     ChainSyncServer $
-      let ChainSyncServer server = chainSyncServerForReader tracer chainDB rdr in
+      let ChainSyncServer server = chainSyncServerForFollower tracer chainDB flr in
       server
 
 -- | A chain sync server.
 --
 -- This is a version of
 -- 'Ouroboros.Network.Protocol.ChainSync.Examples.chainSyncServerExample' that
--- uses a 'chainDB' and a 'Reader' instead of
+-- uses a 'chainDB' and a 'Follower' instead of
 -- 'Ourboros.Network.ChainProducerState.ChainProducerState'.
 --
--- All the hard work is done by the 'Reader's provided by the 'ChainDB'.
+-- All the hard work is done by the 'Follower's provided by the 'ChainDB'.
 --
-chainSyncServerForReader
-    :: forall m blk b.
-       ( IOLike m
-       , HasHeader (Header blk)
-       )
-    => Tracer m (TraceChainSyncServerEvent blk)
-    -> ChainDB m blk
-    -> Reader  m blk (WithPoint blk b)
-    -> ChainSyncServer b (Point blk) (Tip blk) m ()
-chainSyncServerForReader tracer chainDB rdr =
+chainSyncServerForFollower ::
+  forall m blk b.
+    ( IOLike m
```

Comment:
The forall and the parens should both line up with the `T` in `=> Tracer` below.